Clic&Farm-iOS/Controllers/AddInterventionViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  AddInterventionViewController.swift
+//  Clic&Farm-iOS
+//
+//  Created by Guillaume Roux on 30/07/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class AddInterventionViewController: UIViewController {
+    
+    //MARK : Properties
+    
+    @IBOutlet weak var dimView: UIView!
+    @IBOutlet weak var selectCropsView: UIView!
+    @IBOutlet weak var interventionToolsTableView: UITableView!
+    @IBOutlet weak var validateButton: UIButton!
+    @IBOutlet weak var navigationBar: UINavigationBar!
+    @IBOutlet weak var heightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var firstView: UIView!
+    @IBOutlet weak var collapseButton: UIButton!
+    
+    var interventionTools = [NSManagedObject]() {
+        didSet {
+            if interventionTools.count > 0 {
+                interventionToolsTableView.reloadData()
+            } else {
+                interventionToolsTableView.isHidden = true
+            }
+        }
+    }
+    var interventionType: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        UIApplication.shared.statusBarView?.backgroundColor = UIColor(rgb: 0x175FC8)
```

Comment:
Can we extract the colors to constants? Something like `THEME_COLORS['GREY']`/`THEME_COLORS['BLUE']`/... ?